packages/syft/src/syft/service/worker/worker_pool.py

```diff
@@ -79,11 +109,76 @@ class WorkerPool(SyftObject):
 
     __attr_unique__ = ["name"]
     __attr_searchable__ = ["name", "syft_worker_image_id"]
+    __repr_attrs__ = [
+        "name",
+        "syft_worker_image_id",
+        "max_count",
+        "workers",
+        "created_at",
+    ]
 
     name: str
     syft_worker_image_id: UID
+    syft_worker_image_name_tag: Optional[str]
```

Comment:
MAYBE: we could name it `syft_worker_image_tag`  and we could initiliaze with SyftWorkerImageTag class to mantain consistency.

Reply:
Done. Now the `SyftWorkerImage` is passed
![image](https://github.com/OpenMined/PySyft/assets/88959106/492ad7f3-d1c9-4ac7-9ef2-679edf03c8ae)